samples/language/Specializations.qs

```diff
@@ -0,0 +1,33 @@
+/// # Sample
+/// Specializations
+///
+/// # Description
+/// Q# employs symbolic computation that can automatically generate
+/// the corresponding adjoint implementation for a particular body implementation.
+namespace MyQuantumApp {
+
+    /// Implicit
+    operation DoNothing() : Unit 
+        is Adj + Ctl { }
+
+    /// Explicit
```

Comment:
Could you please add a similar level of detail in the comment regarding explicitly defined specializations?

Reply:
Adding a brief comment on each specialization implementation would also be helpful.